local/php/xsendfile.go

```diff
@@ -0,0 +1,21 @@
+package php
+
+import (
+	"net/http"
+	"os"
+)
+
+func (p *Server) processXSendFile(resp *http.Response) (error, bool) {
+	// X-SendFile
+	sendFilename := resp.Header.Get("X-SendFile")
+	if _, err := os.Stat(sendFilename); sendFilename == "" || err != nil {
```

Comment:
no need to stat is the file is empty

Reply:
I kept this part as-is but it's true this is the occasion to improve it! Done